runtime/cudaq/platform/qpu_state.h

```diff
@@ -0,0 +1,153 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#pragma once
+
+#include "common/SimulationState.h"
+#include "cudaq.h"
+#include "cudaq/utils/cudaq_utils.h"
+
+namespace cudaq {
+/// Implementation of `SimulationState` for quantum device backends.
+// The state is represented by a quantum kernel.
+// Quantum state contains all the information we need to replicate a
+// call to kernel that created the state.
```

Comment:
Is all of this intended to be the doxygen comment?

Reply:
fixed, thanks!